src/main/java/ludogorie_soft/reservations_platform_api/config/SecurityConfig.java

```diff
@@ -13,13 +13,20 @@
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.security.web.SecurityFilterChain;
-import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+import org.springframework.web.cors.CorsConfiguration;
+import org.springframework.web.cors.CorsConfigurationSource;
+
+import java.util.Collections;
+import java.util.List;
+
 
 @Configuration
 @EnableMethodSecurity
 @AllArgsConstructor
 public class SecurityConfig {
 
+    public static final String BASE_URL = "http://localhost:3000";
```

Comment:
This should be environment variable because on different environments (live / prod) the address will be different

Reply:
Added environment variable